common/src/mailbox_api.rs

```diff
@@ -276,6 +277,29 @@ impl Default for InvokeDpeReq {
     }
 }
 
+// EXTEND_PCR
+#[repr(C)]
+#[derive(Debug, AsBytes, FromBytes, PartialEq, Eq)]
+pub struct ExtendPcrReq {
+    pub hdr: MailboxReqHeader,
+    pub pcr_idx: u32,
+    pub value: [u8; ExtendPcrReq::DATA_MAX_SIZE], // variable length
```

Comment:
It doesn't seem like this is variable-length? `ExtendPcrReq::read_from()` will fail if the caller doesn't pass in 512 bytes of data.

Reply:
I think we may need a size field here to tell RT how much of `value` to interpret. Also same for StashMeasurementReq